x-pack/plugins/security/public/views/login/components/login_page/login_page.tsx

```diff
@@ -58,15 +58,15 @@ export class LoginPage extends Component<Props, {}> {
               <EuiTitle size="l" className="loginWelcome__title">
                 <h1>
                   <FormattedMessage
-                    id="kbn.login.welcomeTitle"
+                    id="xpack.security.views.login.loginPage.welcomeTitle"
                     defaultMessage="Welcome to Kibana"
                   />
                 </h1>
               </EuiTitle>
               <EuiText size="s" color="subdued" className="loginWelcome__subtitle">
                 <p>
                   <FormattedMessage
-                    id="kbn.login.welcomeDescription"
+                    id="xpack.security.views.login.loginPage.welcomeDescription"
```

Comment:
I would use just `loginPage` in id, without `login`:  `xpack.security.loginPage.welcomeDescription`.